ansible/group_vars/all

```diff
@@ -16,6 +16,7 @@ runtimesManifest:
     default: true
   python:
   - kind: "python"
```

Comment:
kind `python` will resolve to the image `pythonaction`
we could make the image mapping explicit here - or fix it in the ExecManifest. 
making the action to image name mapping explicit here is a good idea and previously suggested by others (@ddragosd @glikson iirc).